Scripts/ants.sh

```diff
@@ -3,20 +3,17 @@
 NUMPARAMS=$#
 
 MAXITERATIONS=30x90x20
-if [[ -z ${ANTSPATH} ]] ; then
-  echo "Environment variable ANTSPATH must be defined"
-  exit 1
-fi
-if [[ ! -f "${ANTSPATH}/ANTS" ]] ; then
-  echo "Cannot find the ANTS program. Please \(re\)define \$ANTSPATH in your environment."
+
+if ! command -v "ANTS" &> /dev/null
```

Comment:
I think you need the square brackets still

Reply:
It works without brackets on my machine. I used the pattern in the accepted answer.

Reply:
```
# Make ANTS disappear from PATH.
$ mv $HOME/.local/ants/bin/ANTS{,.bak}

# Run wrapper script.
$ ./Scripts/ants.sh
Cannot find the ANTS program. Please \(re\)define $PATH in your environment.

# Put it back.
$ mv $HOME/.local/ants/bin/ANTS{.bak,}

# Run wrapper script again.
$ ./Scripts/ants.sh
 USAGE ::  
  sh   ants.sh  ImageDimension  fixed.ext  moving.ext  OPTIONAL-OUTPREFIX   OPTIONAL-max-iterations  OPTIONAL-Labels-In-Fixed-Image-Space-To-Deform-To-Moving-
```